src/keys.ts

```diff
@@ -22,6 +23,7 @@ import { BufferReader } from './bufferReader';
 import { AddressVersion } from './constants';
 import { c32address } from 'c32check';
 import { addressHashModeToVersion, addressToString, addressFromVersionHash } from '.';
+import { sign } from 'crypto';
```

Comment:
This appears to be unused (which is good because the node.js crypto lib can be a pain for browser polyfilling).

Reply:
good catch, I think vscode added this import when I mistyped something